### PR TITLE
Automatically generate a changelog release on tag

### PR DESCRIPTION
**Pull request by jknight-liatrio**
Automagically compile a changelog history and create a github
release after a tag is published and an artifact is created.

#52

### CODE REVIEW

File: .github/workflows/publish.yaml [hunk 1]
```diff
@@ -41,3 +40,18 @@ jobs:
...
             ghcr.io/rode/ui:${{ steps.gitextra.outputs.version }}
+
+      - name: Build Changelog
+        id: github_release
+        uses: mikepenz/release-changelog-builder-action@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Create Release
+        uses: actions/create-release@v1
```
Comment (alexashley) [inline .github/workflows/publish.yaml:51]: Any concerns about this action being unmaintained? https://github.com/actions/create-release#github-action---releases-api

File: .github/workflows/publish.yaml [hunk 2]
```diff
@@ -41,3 +40,18 @@ jobs:
...
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: ${{ github.ref }}
```
Comment (alexashley) [inline .github/workflows/publish.yaml:54]: Is there a way to get the tag name without the `refs/tags` prefix ending up in the release name?

File: .github/workflows/publish.yaml (hunk 1 above)
Comment (jknight-liatrio) [reply .github/workflows/publish.yaml:51]: Good call out. I'll try to find another one.

File: .github/workflows/publish.yaml (hunk 2 above)
Comment (jknight-liatrio) [reply .github/workflows/publish.yaml]: Good call. I removed those explicit references. We'll see what happens now.

Comment (jknight-liatrio) [reply .github/workflows/publish.yaml]: Done. It seems to work properly.

**Pull request merged**